hyfetch/termenv.py

```diff
@@ -88,6 +85,10 @@ def detect_ansi_mode() -> AnsiMode | None:
 
 
 def unix_read_osc(seq: int) -> str:
+    tty = importlib.import_module("tty")
+    signal = importlib.import_module("signal")
+    termios = importlib.import_module("termios")
+    select = importlib.import_module("select").select
```

Comment:
Here you can actually achieve the same effect by just moving the import statements inside the function. Dynamic importing using importlib is only required if the module name is a string that you don't know beforehand.